web/client/plugins/MousePosition.jsx

```diff
@@ -17,10 +17,24 @@ const {changeMousePositionCrs, changeMousePositionState} = require('../actions/m
 
 const selector = createSelector([
     mapSelector || {},
-    (state) => state.mousePosition || {}
-], (map, mousePosition) => ({
+    (state) => state.mousePosition || {},
+    (state) => {
```

Comment:
I don't think this is correct. What we need to make the component work on click is another flag in state.mousePosition (similar to showCenter, something like showClick) and use that to choose between state.mousePosition.position and state.mapInfo.clickPoint